integrations/astra/pyproject.toml

```diff
@@ -51,6 +52,7 @@ dependencies = [
   "coverage[toml]>=6.5",
   "pytest",
   "haystack-pydoc-tools",
+  "astrapy>=0.7.0",
```

Comment:
The jobs depends on the integration already, that depends on `astra`, no need for the extra dep here
```suggestion
```